src/Composer/Installers/RadPHPInstaller.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Composer\Installers;
+
+class RadPHPInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'bundle' => 'bundles/{$name}/'
```

Comment:
Just for interest: what would happen if two authors package will be called the same?